cli_tools_e2e_test/gce_image_import_export/test_suites/import/cli_tests.go

```diff
@@ -372,6 +384,46 @@ func runImageImportWithSubnetWithoutNetworkSpecified(ctx context.Context, testCa
 	runImportTest(ctx, argsMap[testType], testType, testProjectConfig, imageName, logger, testCase)
 }
 
+func runImageImportMissingSubnetworkArg(ctx context.Context, testCase *junitxml.TestCase,
+	logger *log.Logger, testProjectConfig *testconfig.Project, testType utils.CLITestType) {
+
+	suffix := path.RandString(5)
+	imageName := "e2e-test-image-import-bad-network-" + suffix
+
+	argsMap := map[utils.CLITestType][]string{
+		utils.Wrapper: {"-client_id=e2e", fmt.Sprintf("-project=%v", testProjectConfig.TestProjectID),
+			fmt.Sprintf("-image_name=%s", imageName), "-data_disk",
+			fmt.Sprintf("-source_file=gs://%v-test-image/image-file-10g-vmdk", testProjectConfig.TestProjectID),
+			fmt.Sprintf("-network=global/networks/%v-vpc-1", testProjectConfig.TestProjectID),
+			fmt.Sprintf("-zone=%v", testProjectConfig.TestZone),
+			fmt.Sprintf("-execution_id=%v", suffix),
+		},
+	}
+
+	args := argsMap[testType]
+	if args == nil {
+		return
+	}
+
+	utils.RunTestCommandIgnoringError(cmds[testType], args, logger, testCase)
+
+	// Try get shadow disk.
+	shadowDiskName := "shadow-disk-" + suffix
```

Comment:
If the shadow disk's name format changes in the code, and it doesn't get deleted, this test will pass since the new name won't be found.